tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -140,12 +140,27 @@ describe(__filename, () => {
       },
     });
     const root = render({ addon });
+    const link = root.find('.AddonMoreInfo-license-link');
 
     expect(root.find('.AddonMoreInfo-license-title')).toHaveText('License');
-    expect(root.find('.AddonMoreInfo-license-link'))
-      .toHaveProp('children', 'tofulicense');
-    expect(root.find('.AddonMoreInfo-license-link'))
-      .toHaveProp('href', 'http://license.com/');
+    expect(link).toHaveProp('children', 'tofulicense');
+    expect(link).toHaveProp('href', 'http://license.com/');
+  });
+
+  it('does not prefix a license link with the add-ons URL', () => {
+    // See: https://github.com/mozilla/addons-frontend/issues/3339
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      current_version: {
+        ...fakeAddon.current_version,
+        license: { name: 'tofulicense', url: 'www.license.com/' },
+      },
+    });
+    const root = render({ addon });
+    const link = root.find('.AddonMoreInfo-license-link');
+
+    expect(link).toHaveProp('prependClientApp', false);
+    expect(link).toHaveProp('prependLang', false);
```

Comment:
Can we also assert that the link will be a proper link and not something pointing to `www.license.com`?

Reply:
Or do we have such a test in the `Link` component already?

Reply:
Maybe asserting the `href` prop would be enough.

Reply:
Actually the links *aren't* always proper from the API. These will work as browsers fill in the scheme... but eventually we may want to run a migration on the server to convert them and require a scheme in the future 😄

Reply:
I'll file a new issue to add the schema to schema-less-links in `<Link>` though; it's not required but it's nice.

Reply:
Made #3345.

Reply:
💪